dev/CommandBarFlyout/CommandBarFlyout.cpp

```diff
@@ -290,7 +311,7 @@ winrt::Control CommandBarFlyout::CreatePresenter()
                         // We'll only need to do the mid-animation remove/add when the "..." button is
                         // pressed to open/close the overflow. This means we shouldn't do it for AlwaysExpanded
                         // and if there's nothing in the overflow.
-                        if (!AlwaysExpanded() && m_secondaryCommands.Size() > 0)
```

Comment:
Why don't we need the `AlwaysExpanded` check anymore?

Reply:
Because if the overflow popup is not opaque, for instance during the new entrance opacity animation, then even though it is always open, we don't want to have the shadow showing through. So we'll just remove the primary shadow in the always open scenario as well.